Modules/DataCollection/classes/Fields/Reference/class.ilDclReferenceFieldModel.php

```diff
@@ -27,7 +27,7 @@ public function getRecordQuerySortObject($direction = "asc", $sort_by_status = f
 			$n_ref = true;
 		}
 
-		$select_str = ($n_ref) ? " GROUP_CONCAT(stloc_{$this->getId()}_joined.value) AS field_{$this->getId()}" : "stloc_{$this->getId()}_joined.value AS field_{$this->getId()},";
+		$select_str = ($n_ref) ? " ". $ilDB->loadModule('Manager')->getQueryUtils()->stringAgg("stloc_{$this->getId()}_joined.value") . " AS field_{$this->getId()}" : "stloc_{$this->getId()}_joined.value AS field_{$this->getId()},";
```

Comment:
The Manager-Module should not be used in consumer-code outside the setup process or DB-Updates. Consumer code only should use code from ilDBInterface. therefore the ilDBInterface should be extended with your new method. but I'd like to rename this to `groupConcat` instead of `stringAgg`.

I know the `loadModule`mathod is on this interface, too. I just added a `@internal` statement to declare this.